worker/caasapplicationprovisioner/application.go

```diff
@@ -144,13 +144,13 @@ func (a *appWorker) loop() error {
 
 		exists, err := a.broker.OperatorExists(a.name)
 		if err != nil {
-			return errors.Trace(err)
+			return errors.Annotatef(err, "checking if %q has an operator pod due to an upgrade-charm from a pod-spec charm to a sidecar charm", a.name)
```

Comment:
"... due to upgrading the charm from a pod-spec charm to ..."

Reply:
hmm, this was copied from the comment above..